hacks/server/test_tunnel.sh

```
@@ -0,0 +1,287 @@
+
+log_info "Setting up test environment..."
+# Create namespace if it doesn't exist
+kubectl create namespace $NAMESPACE 2>/dev/null || true
+log_success "Namespace $NAMESPACE ready"
+
+# Clean up any previous instances
+log_info "Cleaning up any previous instances..."
+kubectl delete service kftray-server -n $NAMESPACE 2>/dev/null || true
+kubectl delete pod kftray-server --force --grace-period=0 -n $NAMESPACE 2>/dev/null || true
+kubectl delete pod curl-test --force --grace-period=0 -n $NAMESPACE 2>/dev/null || true
+kill $(lsof -t -i:8085) 2>/dev/null || true
```

Shellcheck (0.9.0-1)

[warning] Quote this to prevent word splitting.

(SC2046)